src/javascript/app/pages/trade/starttime.js

```diff
@@ -102,6 +102,11 @@ const StartDates = (() => {
                 Durations.onStartDateChange(first);
             }
         } else {
+            if (start_dates && start_dates.has_spot) {
+                const now_option = createElement('option', { value: 'now', text: localize('Now') });
+                CommonIndependent.getStartDateNode().appendChild(now_option);
+                has_now = 1;
+            }
```

Comment:
the problem was that time picker gets starting date from the `#date_start`, but it's value was `''` since it was an empty selector with no options